docs/ensuring-repro/managing-software/index.md

```diff
@@ -16,3 +16,13 @@ We specifically recommend these two package managers to specify module-specific
 Conda can also be used to manage standalone software packages that do not depend on a specific language.
 
 When you [create a module with `create-analysis-module.py`](../../contributing-to-analyses/analysis-modules/creating-a-module.md), you can use one (or more!) of several [flags](../../contributing-to-analyses/analysis-modules/creating-a-module.md#module-creation-script-flags) to establish your module with an initialized software environment.
+
+## Updating software environments
+
+As analysis modules mature, the Data Lab will activate several [GitHub Action workflows](../workflows/index.md) that ensure module reproducibility.
+Once these workflows are activated, you will need to file _two PRs_ every time you wish to update your software environment:
```

Comment:
```suggestion
As analysis modules mature, the Data Lab will implement several [GitHub Action workflows](../workflows/index.md) that ensure module reproducibility.
Once these workflows are implemented, you will need to file _two PRs_ every time you wish to update your software environment:
```
I don't think activated is the right word choice here.

Reply:
Again I would mention that an example of this is using a new package.

Reply:
"Activate" is actually the language we have been using, for example see here: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/f4810f8abb007c57d0622fa028083c57cd9804d4/docs/ensuring-repro/workflows/run-module-gha.md?plain=1#L31-L33

But I can make it clearer that the files will already have been created, they just won't be turned on (activated) yet and [point to these docs too for reference](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/f4810f8abb007c57d0622fa028083c57cd9804d4/docs/contributing-to-analyses/analysis-modules/creating-a-module.md?plain=1#L59).